front/pages/api/w/[wId]/workspace-usage.ts

```diff
@@ -86,7 +86,7 @@ async function handler(
           case "all":
             return {
               startDate: new Date("2020-01-01"),
-              endDate: new Date(),
+              endDate: endOfMonth(new Date()),
```

Comment:
unrelated change, sorry. But I realized workspace-usage was not reporting current day usage in "all time" mode.

Reply:
ack